content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -441,19 +626,88 @@ issued to the public administrations must contain:
-     (DatiConvenzione), **2.1.5** (Date Ricezione), or **2.1.6** (Dati Fatture Collegate)
-     information blocks. These correspond to the elements named :guilabel:`CodiceCUP` and
-     :guilabel:`CodiceCIG` of the electronic invoice **XML** file, whose table can be found on the
-     government `website <http://www.fatturapa.gov.it/>`_.
-   - The :abbr:`CUU (Codice Univoco Ufficio)` must be included in the electronic invoice
-     corresponding to the element **1.1.4** (:guilabel:`CodiceDestinario`).
+   - The :abbr:`CUP (Codice Unico di Progetto)` and the :abbr:`CIG (Codice Identificativo Gara)`
+     must be included in one of the `DatiOrdineAcquisto`, `DatiContratto`, `DatiConvenzione`,
+     `DateRicezione`, or `DatiFattureCollegate` **XML** tags.
+     These correspond to the elements named :guilabel:`CodiceCUP` and :guilabel:`CodiceCIG` of the
+     electronic invoice **XML** file, whose table can be found on the government `website
+     <http://www.fatturapa.gov.it/>`_.
+
+.. _italy/split-payment:
+
+Split Payment
+-------------
+
+The :guilabel:`Split Payment` mechanism behaves much like :ref:`italy/reverse-charge` and it's
+been used for the same Tax Evasion purpose.
+
+When an Italian company bills a :abbr:`PA (Public Administration)` business (for example they're
+selling cleaning services for a public building) the :abbr:`PA (Public Administration)` business
+will send the VAT to the Tax Agency themselves, and the vendor just has to use an appropriate
+:guilabel.
+
+In the Italian Localization we have configured a special :guilabel:`Fiscal Position` to deal with
+partners belonging to the :abbr:`PA (Public Administration)`.
+
+.. figure:: italy/italy-split-payment-fp.png
+   :align: center
+   :alt: Public Administration Fiscal Position
+
+   Public Administration Fiscal Position
+
+Process
+-------
+
+.. _italy/digital-signature:
+
+1. Digital qualified signature
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For invoices and bills intended to the **Pubblica Amministrazione (B2G)**, a **Digital Qualified
+Signature** is required for all files sent through the :abbr:`SdI (Sistema di Interscambio)`. The
+**XML** file must be certified using a certificate that is either:
+
+- a **smart card**;
+- a **USB token**;
+- an :abbr:`HSM (Hardware Security Module)`.
+
+.. warning::
+   Odoo **cannot** digitally sign documents for you. When we detect a :guilabel:`Codice
+   Destinatario` that is 6 digits long, then the :abbr:`EDI (Electronic Data Interchange)` process
+   stops, the invoice is set on the :guilabel:`Requires user signature` state. You can download the
+   document in **XML**, sign it with any :guilabel:`Digital Qualified Signature` provider's external
+   program and send it through the :abbr:`AdE (Agenzia Delle Entrate)` portal.
+
+    .. figure:: italy/italy-edi-requires-user-signature.png
+       :align: center
+       :alt: EDI Request User Signature State
+
+       EDI Request User Signature State
+
+2. Acceptance or Refusal
```

Comment:
Acceptance and Refusal

Reply:
Why? They are mutually exclusive.

Reply:
my bad, what I meant is to remove the bullet point not to change it with and